### ISSUE TITLE
Add keyboard interaction accessibility tests

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#3953]
**Describe the issue. What is the expected and unexpected behavior?**
We should write cypress keyboard interaction accessibility tests for our components in order to prevent future regressions, focus should be given to the most popular components: 
- [x] Button
- [ ] Modal
- [ ] Card
- [ ] Form
- [ ] Alert
- [ ] Empty State
- [ ] Tooltip 

(All dropdown related components captured in a separate issue: #3108)

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
This would be an enhancement to cut down on manual testing time.

**Comment by thatblindgeye** [comment]
Due to inactivity I'm going to close this out for now, but moved to Parking Lot to keep in mind that it's something to revisit when we have the bandwidth.
